test/Microsoft.OpenAPI.OData.Reader.Tests/Generator/OpenApiSchemaGeneratorTests.cs

```diff
@@ -488,7 +534,7 @@ public void CreatePropertySchemaForNullableEnumPropertyReturnSchema(OpenApiSpecV
         public void GetDerivedTypesReferenceSchemaReturnsDerivedTypesReferencesInSchemaIfExist()
         {
             // Arrange
-            IEdmModel edmModel = EdmModelHelper.GraphBetaModel;
+            IEdmModel edmModel = OData.Tests.EdmModelHelper.GraphBetaModel;
```

Comment:
> OData.Tests. [](start = 32, length = 13)

the above change use "Test." prefix, so, shall we use the same here?
Or do we remove them?

Reply:
Remove - this was just a hangover from a namespace change I was fiddling with and forgot to remove.